### PR TITLE
Upgrade SmallRye Health to 3.2.0

### PR DESCRIPTION
**Pull request by xstefank**

### CODE REVIEW

**Review by gsmet — APPROVED** [review]

File: bom/application/pom.xml [hunk 1]
```diff
@@ -3302,6 +3302,12 @@
                 <groupId>io.smallrye</groupId>
                 <artifactId>smallrye-health</artifactId>
                 <version>${smallrye-health.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>jakarta.json</groupId>
+                        <artifactId>jakarta.json-api</artifactId>
```
Comment (gsmet) [inline bom/application/pom.xml:3308]: Question, do you need JSON-P around or is it just optional?

**Comment by gsmet** [comment]
I requested another CI run as I would have to get the MP TCK tests green (it's CP who made the build stall but we didn't have a clear report of the Health TCK status).

File: bom/application/pom.xml (hunk 1 above)
Comment (xstefank) [reply bom/application/pom.xml:3308]: yes, we also now have a non-CDI plain option of using sr-health so I needed to ship this too, unfortunately.

Comment (gsmet) [reply bom/application/pom.xml:3308]: Ah OK, I just checked and the dependency is already there. Fine then.

**Pull request merged**